tests/conftest.py

```diff
@@ -103,9 +103,25 @@ def fill_basic_db(db):
 def new_test_db(uri):
     dbname = uri[uri.rindex("/") + 1 :]
     dbname_base = DATABASE_URI_BASE[DATABASE_URI_BASE.rindex("/") + 1 :]
-    dump_args = ["mysqldump", "-h", "db", "-u", "root", f"-p{mysql_root_password}", dbname_base]
-    load_args = ["mysql", "-h", "db", "-u", "root", f"-p{mysql_root_password}", dbname]
-    proc1 = subprocess.run(dump_args, capture_output=True)
```

Comment:
This row is deleted. Can you add info here about why if this was intentional?

Reply:
It wasn't doing anything. My best guess is that it was a testing thing that was left there.

It's the next subprocess which pipes the output to be read by the load